src/components/Tooltip/StyledTooltip.js

```diff
@@ -77,14 +77,24 @@ const Tip = styled.span`
     font-size: 0.875rem;
     line-height: 1rem;
     white-space: pre-wrap;
-    background-color: #FFF;
+    background-color: ${({ theme }) => theme.colors.tooltip.tipBackground};
     box-shadow: 0 1em 2em -0.5em rgba(0, 0, 0, 0.35);
   }
 `
 
 const Icon = styled(Anchor)`
+  font-family: 'Times-Italic', 'Times Italic', 'Times', sans-serif;
   font-style: italic;
-  padding: 0.375rem;
+  font-weight: 400;
+  color: ${({ theme }) => theme.colors.tooltip.iconColor};
+  margin: 0;
+  border: 1px solid black;
```

Comment:
We should use the MAS green for the border of the icon.

Reply:
well spotted @davidtrussler. Let us know when you've made this small change @andreferreiradlw :)

Reply:
I've adjusted this now to MAS green on the theme and made this dynamic